PAO-Project/src/com/company/Main.java

```diff
@@ -0,0 +1,12 @@
+package com.company;
+
+import Services.App_Service;
+import java.text.ParseException;
+
+public class Main {
+
+    public static void main(String[] args) throws ParseException {
```

Comment:
![image](https://user-images.githubusercontent.com/67580205/161395047-b7df050b-5f3e-431f-b79f-2aafc55fc90d.png)
selectarea meniului principal mi-l afiseaza de 2 ori

Reply:
![image](https://user-images.githubusercontent.com/67580205/161395090-692b7e5a-df6e-4b22-b173-39f8760c546a.png)
nu te astepta ca userul sa bage ce te astepti tu, userii vor gasi mereu o modalitate de a o da in bara

Reply:
![image](https://user-images.githubusercontent.com/67580205/161395133-083839dd-b3ec-421a-ad77-e7ee6f6bb54d.png)
aplicatia crapa daca noi cautam dupa un id care nu exista